ext/standard/tests/serialize/sleep_uninitialized_typed_prop.phpt

```diff
@@ -1,15 +1,16 @@
 --TEST--
-Referencing an uninitialized typed property in __sleep() should result in Error
+Referencing an uninitialized typed property in __sleep() should be skipped
 --FILE--
 <?php
 
 class Test {
+    public ?int $w = 123;
```

Comment:
This looks like it will do something I wouldn't expect for an unset `public array $notNullable = ['someValue'];`

```php
class Example {
    public array $notNullable = ['someValue'];
    public function __sleep() { return ['notNullable']; }  // also ['notNullable', 'notNullable'] should emit a notice.
}
$x = new Example();
unset($x->notNullable);
serialize($x);  // I'd expect this to throw because unserialize(serialize($x)) != $x, but this code looks like it won't
```

```c
			if (Z_ISUNDEF_P(prop) || !ZEND_TYPE_ALLOW_NULL(prop_info->type)) {
				return SUCCESS;
```

Also, further behaviors would probably be more readable in individual test files.